### PR TITLE
Initial pass for About page

### PR DESCRIPTION
**Pull request by ryelle**
This PR moves the copy from [the doc](https://docs.google.com/document/d/1gnoOTQ8RuT_NTEgt3RtAc8KiwsAw73C4wEABNWuJADo/edit#) into code, using [the design from Figma](https://www.figma.com/file/ugeB3wyCgJGjwcTDxqrAXq/About-Page-6.1?node-id=1273%3A15661).

~The header is still the 6.0 version, last I heard the design here is still in progress.~ Header is still in progress, but I've updated the PR to remove the "6.0" image.

All the content and links should be in place, but none of the images have been exported yet (so they're red circles). The final images aren't necessary for RC.

I changed the layout of the last section in the icon grid — "WordPress 6.1 performance highlights" was very long, so I combined 3 smaller sections to line them up better (bonus, those were missing icons, now they don't need any).

One last section, "Block template parts in classic themes", had a repeat icon, so that's also been placeholder-ified with a red circle.

This is missing translator comments for the strings.

about

Trac ticket: https://core.trac.wordpress.org/ticket/56357

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

File: src/wp-admin/about.php [hunk 1]
```diff
@@ -49,214 +49,398 @@
...
+				</ul>
 			</div>
 		</div>
 
-		<hr class="is-large" />
+		<hr />
 
 		<div class="about__section has-2-columns is-wider-right">
 			<div class="column about__image is-vertically-aligned-top">
 				<a href="https://www.youtube.com/watch?v=oe452WcY7fA">
```
Comment (sabernhardt) [inline src/wp-admin/about.php]: Reminder: this video link will need to be updated, too.

File: src/wp-admin/about.php [hunk 2]
```diff
@@ -49,214 +49,398 @@
...
+			<div class="column">
+				<div class="about__image">
+					<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false">
+						<rect width="48" height="48" rx="4" fill="#1E1E1E"/>
+						<path fill-rule="evenodd" clip-rule="evenodd" d="M23.2639 25.5896L18.7107 30.2783L17.6346 29.2333L22.1471 24.5865L16.0056 24.5598L16.0121 23.0598L22.2329 23.0869L17.9693 18.6964L19.0454 17.6514L23.2449 21.9759L23.2136 16.0062L24.7136 15.9983L24.7446 21.9117L28.8818 17.6514L29.9579 18.6964L25.6798 23.1019L32.1563 23.13L32.1498 24.63L25.7955 24.6024L30.2926 29.2333L29.2165 30.2783L24.7644 25.6937L24.7983 32.1497L23.2983 32.1576L23.2639 25.5896Z" fill="white"/>
+					</svg>
+				</div>
+				<h3 class="is-smaller-heading"><?php _e( 'Other notes of interest' ); ?></h3>
+				<p><?php _e( '6.1 includes a new time-to-read feature showing content authors the approximate time-to-read values for pages, posts, and custom post types.' ); ?></p>
+				<p><?php _e( 'The site tagline is empty by default in new sites but can be modified in General Settings.' ); ?></p>
```
Comment (sabernhardt) [inline src/wp-admin/about.php]: When I suggested mentioning the site tagline, I wanted a link to the "General Settings" page (if the user has the proper permissions).

```
<?php 
	printf(
		/* translators: %s: "General Settings" admin page title, linked to the page if the user can edit options. */
		__( 'The site tagline is empty by default in new sites but can be modified in %s.' ),
		current_user_can( 'manage_options' ) ? '<a href="' . esc_url( admin_url( 'options-general.php' ) ) . '">' . __( 'General Settings' ) . '</a>' : __( 'General Settings' )
	);
?>
```

Comment (ryelle) [reply src/wp-admin/about.php]: Added in https://github.com/WordPress/wordpress-develop/pull/3433/commits/b2d1dbc7082a2950abd7caac51f97b5f22a88496 👍🏻

File: src/wp-admin/about.php [hunk 3]
```diff
@@ -49,214 +49,421 @@
...
-					<?php _e( 'Style Switching' ); ?>
-				</h3>
+		<div class="about__section has-2-columns">
+			<div class="column is-vertically-aligned-center">
+				<h3><?php _e( 'A better creator experience with refined and additional templates' ); ?></h3>
 				<p>
-					<?php _e( 'Block themes now include the option to contain multiple style variations. This expands the new Style system even further and enables shortcuts to switch the look and feel of your site all within a single theme. You can change both the available settings, like the font weight, and the style options, like the default color palette. Change the look and feel of your site with just a few clicks.' ); ?>
+					<?php
+					printf(
+						/* translators: %1$s: Link to template options dev note, %2$s: Link to template creation dev note. */
```
Comment (sabernhardt) [inline src/wp-admin/about.php]: `/* translators: 1: Link to template options dev note, 2: Link to template creation dev note. */`

Translator comments only need the numbers for placeholders, like in the author template:

https://github.com/WordPress/wordpress-develop/blob/14e51e1f16b2561f00eb9f3f2bfbf17700b7ffd9/src/wp-includes/author-template.php#L511-L512

Comment (ryelle) [reply src/wp-admin/about.php]: Ah, thanks

**Comment by ryelle** [comment]
Latest changes & copy

about

**Comment by ryelle** [comment]
Asked about the design items in slack: https://wordpress.slack.com/archives/C02S78ZAL/p1665506685176209

**Comment by ryelle** [comment]
There are still remaining design issues but we can address those post RC — it's nothing that would change the strings.

**Comment by ryelle** [comment]
Latest screenshot:

Screen Shot 2022-10-11 at 13 47 57

**Comment by ryelle** [comment]
committed in [r54499](https://core.trac.wordpress.org/changeset/54499)

**Comment by DanSoschin** [comment]
Per some recent milestone discovery, there's a feature highlighted on the About Page that will not be part of 6.1. Therefore, we will need to remove the entire bullet from the About Page copy.

> Header and footer patterns for every theme
> Explore these block patterns, making [header](https://learn.wordpress.org/tutorial/customizing-a-header-with-patterns/) and [footer](https://learn.wordpress.org/tutorial/customizing-a-footer-with-patterns/) creation more efficient.

**Comment by DanSoschin** [comment]
Additionally, we need to remove the following sentence from the copy, because the number value isn't an accurate representation of the material changes (fixes and enhancements). 

> WordPress 6.1 includes more than 2,000 updates.

I will sponsor a discussion separately from this ticket as to how best count and communicate product update (bug fixes and enhancement) counts in communications for future assets.

**Pull request closed without merging**